fg_module.py

```diff
@@ -0,0 +1,28 @@
+import os
+import pygame
+
+import main
+
+# Ground
+ground = pygame.image.load(os.path.join('Utils/Pics/Foreground','ground.png')).convert_alpha()
+flipped_ground = pygame.transform.flip(ground, True, False)
+ground_x = 0
+ground_y = main.win.get_height()-ground.get_height() 
+ground_width = ground.get_width() - 5	# To prevent glitches in background movement...yet to find an optimal solution
```

Comment:
You should comment something like 'WIP', such that, in the future, you can simply search WIP and fix all these improving bits!